app/assets/stylesheets/elements/_show-on-focus-within.scss

```diff
@@ -0,0 +1,7 @@
+.show-on-focus-within__content {
+  display: none;
+
+  .show-on-focus-within:focus-within & {
```

Comment:
That would actually be backwards. `focus-within` activates when there is focus on any element within that element, so `show-on-focus-within` is actually the parent element.

Reply:
Right, the `__target` indicates that the specified element is the thing that needs to be focused for the `__content` to be shown

Reply:
That doesn't work in this case, as you can focus other things inside the element, in this case you might move the focus from the input to the buttons on the bubble itself, at which point it would disappear unless those also have the class. We want to capture focus on everything inside the element

Reply:
Ah I see. Then I think the current naming convention is fine.